client/src/app/page.module.css

```diff
@@ -4,8 +4,62 @@
     justify-content: space-between;
     align-items: center;
     padding: 6rem;
-    min-height: 100vh;
+    min-height: 100%;
     /* background-color: #A9CBBC; */
   }
-  
+
+.landing {
+    position: absolute;
+    top: 130px;
+    left: 50px;
+}
+
+.welcome {    
+    color: white;
+    font-size: 90px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;   
+}
+
+.message {
+    color: white;
+    font-weight: 500;
+    line-height: normal;
+    font-size: 30px;
+    margin-top: 60px;
+    margin-left: 50px;
+    width: 700px;
```

Comment:
Try to avoid adding width with pixels to a container if you can since we want to make sure the website can be responsive!